### DISCUSSION TITLE
part-ii/2021-119/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#392] [Question comments]
Part II 2021 Principles of Statistics: Paper 3, Section II, J 

https://questions.tripos.org/part-ii/2021-119/

**Comment by ArthurConmy** [comment]
<details>
<summary>a) </summary>

$$ \hat{\beta} = \frac{ n\alpha }{ x_1+...+x_n }. $$

By CLT $ \sqrt{n}( \bar{X}_n - \frac{\alpha}{\beta} ) \to^d N( 0, \alpha / \beta^2 ) $.

By the delta method with $ \Phi(x) = \alpha / x $, we get

$$ \boxed{ \sqrt{n}( \hat{\beta} - \beta ) \to^d N(0, \beta^2 / \alpha) } $$

and this agrees with the asymptotic normality result.
</details>

<details>
<summary>b)</summary>

We need an estimator of $ \beta^2 / \alpha $ in order to use a symmetric set around $\hat{\beta}$.

By SLLN $(X_1 + ... + X_n)/n \to \alpha / \beta$ almost surely and so (doing things like the proof of credible sets being asymptotically confidence sets throughout), choose the interval $ I_n := [ \hat{\beta}_n - R_n , \hat{\beta}_n + R_n ] $ where $R_n$ is chosen such that $ \mathbb{ P}[ |Z| \le R_n ]=1-\gamma$ where $Z \sim N(0, \alpha / S_n^2)$. Then it suffices to show that $R_n \to \Phi_0^{-1}( 1 - \gamma )$ almost surely where $\Phi_0( z ) = \mathbb{ P}[ |\tilde{Z}| \le z ]$ and $\tilde{Z} \sim N(0, \beta^2 / \alpha)$ and finish by Slutsky as in the notes.

</details> 

c) is then bookwork, too.
